include/DetectorModule.hh

```diff
@@ -146,6 +147,9 @@ public:
 
   Property<bool, Default> removeModule;
 
+  Property<double, Default> twistAng;
```

Comment:
yawAngle
Do not hesitate to not abbreviate the names, it makes the reading easier, and eventually the most time is taken understanding the code, while typing is an infinitesimal part.